src/tests/unit/paginator.test.js

```diff
@@ -1,8 +1,10 @@
-const appRoot = require('app-root-path');
-const { assert } = require('chai');
+import { assert } from 'chai';
 
-const { paginate } = appRoot.require('/utils/paginator');
-const rows = appRoot.require('/tests/unit/mock-data.json').pets;
+import pets from './mock-data';
+
+import { paginate } from 'utils/paginator';
+
+const rows = pets.pets;
```

Comment:
I could be wrong, but I guess line 3 could become:

```js
import { pets: { rows } } from './mock-data';
```

Reply:
Apparently you can't do a nested destructure on an ES6 import: https://github.com/babel/babel/issues/4996 so the extra statement is required.

Reply:
From our discussion, destructuring is possible by doing something like this `import { pets as rows } from './mock-data'` since it isn't nested. Thanks for pointing this out